package.json

```diff
@@ -14,7 +14,7 @@
     "start": "node scripts/start",
     "test": "jest",
     "test:api": "node scripts/test/api",
-    "itest": "jest --config=itest/config.json --runInBand --verbose",
```

Comment:
Will this still show the same detail/verbosity when tests fail?

Reply:
Yes, it shows the failures like a normal jest runner.